egs/tedlium/s5/local/nnet3/run_tdnn.sh

```diff
@@ -0,0 +1,113 @@
+#!/bin/bash
+
+# this is the standard "tdnn" system, built in nnet3; it's what we use to
+# call multi-splice.
+
+# Results: 
+# %WER 15.3 | 507 17792 | 87.4 9.0 3.6 2.7 15.3 90.1 | -0.081 | exp/nnet3/tdnn_sp/decode_dev/score_10_0.5/ctm.filt.filt.sys
+# %WER 13.9 | 507 17792 | 88.4 8.0 3.6 2.3 13.9 85.8 | -0.164 | exp/nnet3/tdnn_sp/decode_dev_rescore/score_10_0.5/ctm.filt.filt.sys
+# %WER 13.8 | 1155 27512 | 88.5 8.7 2.7 2.3 13.8 84.2 | -0.076 | exp/nnet3/tdnn_sp/decode_test/score_10_0.0/ctm.filt.filt.sys
+# %WER 12.5 | 1155 27512 | 89.6 7.7 2.6 2.1 12.5 81.5 | -0.133 | exp/nnet3/tdnn_sp/decode_test_rescore/score_10_0.0/ctm.filt.filt.sys
+
+# At this script level we don't support not running on GPU, as it would be painfully slow.
+# If you want to run without GPU you'd have to call train_tdnn.sh with --gpu false,
+# --num-threads 16 and --minibatch-size 128.
+
+stage=9
+affix=
+train_stage=-10
+speed_perturb=true
```

Comment:
Only the nnet experiment dir name, or all related experiment dirs 
including ali, lats and chain tree?

在 2016/4/7 13:07, Vijayaditya Peddinti 写道:
>
> In egs/tedlium/s5/local/nnet3/run_tdnn.sh 
> <https://github.com/kaldi-asr/kaldi/pull/644#discussion_r58819713>:
>
> > +# call multi-splice.
> > +
> > +# Results:
> > +# %WER 15.3 | 507 17792 | 87.4 9.0 3.6 2.7 15.3 90.1 | -0.081 | exp/nnet3/tdnn_sp/decode_dev/score_10_0.5/ctm.filt.filt.sys
> > +# %WER 13.9 | 507 17792 | 88.4 8.0 3.6 2.3 13.9 85.8 | -0.164 | exp/nnet3/tdnn_sp/decode_dev_rescore/score_10_0.5/ctm.filt.filt.sys
> > +# %WER 13.8 | 1155 27512 | 88.5 8.7 2.7 2.3 13.8 84.2 | -0.076 | exp/nnet3/tdnn_sp/decode_test/score_10_0.0/ctm.filt.filt.sys
> > +# %WER 12.5 | 1155 27512 | 89.6 7.7 2.6 2.1 12.5 81.5 | -0.133 | exp/nnet3/tdnn_sp/decode_test_rescore/score_10_0.0/ctm.filt.filt.sys
> > +
> > +# At this script level we don't support not running on GPU, as it would be painfully slow.
> > +# If you want to run without GPU you'd have to call train_tdnn.sh with --gpu false,
> > +# --num-threads 16 and --minibatch-size 128.
> > +
> > +stage=9
> > +affix=
> > +train_stage=-10
> > +speed_perturb=true
>
> since we are rewriting all the recipes now would be the time to drop 
> it and still maintain consistency in naming across all recipes. So 
> @freewym <https://github.com/freewym> @galv <https://github.com/galv> 
> @naxingyu <https://github.com/naxingyu> you can drop the _sp suffix 
> from the experiment directory name.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub 
> <https://github.com/kaldi-asr/kaldi/pull/644/files/036396efc35ad2cdb5b2039fdfce6b8e9f3d5786#r58819713>
>